src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -589,6 +589,20 @@
     <Field Name="ConstantValueOpt" Type="ConstantValue?"/>
   </Node>
 
+  <!--
+  Default expressions can convert to the target type.
+  Once converted to a target type, they cannot be target-typed again.
+  The Converted default expression is one which has been already converted to a target type.
+  Converted default expressions always has a type.
+  Does not survive initial binding.
```

Comment:
>Does not survive initial binding. [](start = 2, length = 33)

Is this accurate? It looks like CodeGen is prepared to deal with the node.

Reply:
Oops. The comment should be on `BoundDefaultExpression`. I'll move.

---
In reply to: [314033534](https://github.com/dotnet/roslyn/pull/37596#discussion_r314033534) [](ancestors = 314033534)